king_phisher/server/plugins.py

```diff
@@ -134,24 +134,25 @@ def __init__(self, config):
 		self.config = config
 		path = self._get_path()
 		self._server = None
-		super(ServerPluginManager, self).__init__(path, (config,))
+		super(ServerPluginManager, self).__init__(path, (config,), config.get_if_exists('server.plugin_library_path'))
 		for plugin in config.get_if_exists('server.plugins', {}).keys():
 			# load the plugin
-			try:
-				self.load(plugin)
-			except Exception:
-				self.logger.critical('failed to load plugin: ' + plugin, exc_info=True)
-				raise errors.KingPhisherPluginError(plugin, 'failed to load')
+			self._load_plugin(plugin)
 			# check compatibility
 			klass = self[plugin]
 			for req_type, req_value, req_met in klass.compatibility:
 				req_type = req_type.lower()
 				if req_met:
 					self.logger.debug("plugin '{0}' requirement {1} ({2}) met".format(plugin, req_type, req_value))
 					continue
-				self.logger.warning("plugin '{0}' unmet requirement {1} ({2})".format(plugin, req_type, req_value))
-				raise errors.KingPhisherPluginError(plugin, 'failed to meet requirement: ' + req_type)
-			# enable the plugin
+				if req_type == 'Required Package' or not req_met:
```

Comment:
With the line `req_type = req_type.lower()` above, the `req_type == 'Required Package'` portion of this expression will never evaluate to True.